.env.example

```diff
@@ -0,0 +1,3 @@
+ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
```

Comment:
Cool there's a .env file here. I'm guessing the install instructions should say to copy this to `.env`. And then what, do I need to generate some values?